core/NameHash.h

```diff
@@ -47,7 +47,7 @@ struct GlobalStateHash {
 
 struct UsageHash {
     std::vector<core::NameHash> sends;
-    std::vector<core::NameHash> constants;
+    std::vector<core::NameHash> symbolNames;
```

Comment:
Nit: I think since this isn't storing names, but hashes, that `symbolsNames` is not the best.  Maybe `symbols`...which is still not quite right but is at least parallel to `sends`, above?

Reply:
Fine with me!